### PR TITLE
Yeoman-welcome revamp

### PR DESCRIPTION
**Pull request by KoolTheba**
### Changes:
- Fix missing description in Ava test
- ES6 refactor `index.js`
- Chalk and Node engine upgrade version
- Node versions upgraded in Travis

Thanks in advance @UlisesGascon and @sindresorhus for the code review ^.^

### CODE REVIEW

**Review by UlisesGascon — APPROVED** [review]
looks fine for me! 

@sindresorhus should we update the version in `package.json` also?

File: index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
 'use strict';
-var chalk = require('chalk');
+const chalk = require('chalk'),
+    r = chalk.red,
+    yB = chalk.yellow.bold,
+    y = chalk.yellow;
```
Comment (sindresorhus) [inline index.js]: This makes the code a lot less readable. All the changes in this file should be reverted.

File: test.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import test from 'ava';
-import x from './';
+import welcomeText from './index';
```
Comment (sindresorhus) [inline test.js]:
```suggestion
import welcomeText from '.';
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Thanks for working on this :)

**Comment by sindresorhus** [comment]
Bump

File: index.js (hunk 1 above)
Comment (KoolTheba) [reply index.js]: Thanks for your feedback @sindresorhus 
I'll revert it asap ^^

Comment (KoolTheba) [reply index.js]: Fixed in https://github.com/yeoman/yeoman-welcome/pull/2/commits/bb4fd964b553a6d525bced121216ee4be61e6169

**Comment by KoolTheba** [comment]
@sindresorhus thank you for checking, I think is finally ready :rocket:

**Comment by SBoudrias** [comment]
Thanks!

**Pull request merged**